src/extra/AutoUI/index.tsx

```diff
@@ -478,17 +497,17 @@ export const getColumnsFromSchema = <T extends AutoUIBaseResource<T>>({
 
 			const widgetSchema = { ...val, title: undefined };
 			return {
-				title: val.title || key,
+				title: getSchemaTitle(val, key, refScheme?.[0]),
 				field: key,
 				// This is used for storing columns and views
-				key,
+				key: `${key}_${index}`,
```

Comment:
Why did we need to make this more unique?

Reply:
This is because some properties may come from the same object, as we now use the refScheme as an array. eg:

```
is_for__device_type: {
					title: 'Device type',
					type: 'array',
					description:
						'{"x-ref-scheme": ["name", "is_of__cpu_architecture[0].slug"]}',
					items: {
						type: 'array',
						properties: {
							name: {
								title: 'Device type',
								format: 'device-type',
								type: 'string',
							},
							is_of__cpu_architecture: {
								type: 'array',
								items: {
									type: 'array',
									properties: {
										slug: {
											title: 'Architecture',
											type: 'string',
										},
									},
								},
							},
						},
					},
				},
```

In this case, the key would be for both items `is_for__device_type` which is not something we want for the filterModal select